### DISCUSSION TITLE
The Google Calendar on your Google Workspace account is not linked to your Kimai calendar.

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4758] [Plugins]
https://www.kimai.org/documentation/local-yaml.html

Referring to the Google Calendar integration in the kimai guide configuration file above, I have set up all schedule details view in the access rights of the Google Calendar api key and Google Gallinder as below in /kimai/config/packages/local.yaml. I can see the holiday schedules in the kimai calendar, but I can't see the company group schedules.

Kimai:
    Calendar:
        Google:
            api_key: 'hidden api key'
            Sources:
                Holidays:
                    id: 'en.ddd#holiday@group.v.calendar.google.com'
                    colour: '#ccc'
                Company:
                    id: 'company.com_adasd@group.calendar.google.com'
                    colour: '#cc0000'

Do I need to change anything in the Google Calendar setting of my Google Workspace account or in /kimai/config/packages/local.yaml?

**Comment by seleondevel** [comment]
@kevinpapst  I really want to solve this problem, please help me.

**Reply by kevinpapst** [reply]
I think you misunderstand "free" vs "open source". 
I give you a software for free. 
That does not mean I give you my time for free.
If you need urgent help for for your project, I am available for paid support.

**Reply by kevinpapst** [reply]
Posting stuff and then deleting the answer is lame, closing
